buildSrc/src/main/java/org/eclipse/dataspaceconnector/gradle/DependencyRulesPlugin.java

```diff
@@ -25,9 +25,8 @@ public class DependencyRulesPlugin implements Plugin<Project> {
 
     @Override
     public void apply(Project project) {
-        var extension = project.getExtensions().create(
-                "dataspaceconnectorplugin", DependencyRulesPluginExtension.class
-        );
+        var extension = project.getExtensions()
+                .create("dataspaceconnectorplugin", DependencyRulesPluginExtension.class);
```

Comment:
While you are at it, and if you don't mind, you could rename the extension as well to "dependencyrulespluginextension".